cdk-lib/viewer-stacks/viewer-nodes-stack.ts

```diff
@@ -27,9 +28,10 @@ export class ViewerNodesStack extends cdk.Stack {
 
         // Some configuration values
         const viewerPort = 8005; // Arkime default
-        const viewerPass = new secretsmanager.Secret(this, "ViewerPassword", {
+        const viewerUser = "admin";
+        const viewerPass = new secretsmanager.Secret(this, 'ViewerPassword', {
             generateSecretString: {
-                excludeCharacters: '\\$:()[]&\'\"<>`|;*?# ' // Characters likely to cause problems in shells
+                excludeCharacters: '\\$:()[]&\'\'<>`|;*?# ' // Characters likely to cause problems in shells
```

Comment:
Shouldn't this still have \\"

Reply:
Ah, yep - it should.  Thanks, will update.